src/sagas/storage.ts

```diff
@@ -0,0 +1,242 @@
+import { SagaIterator } from 'redux-saga';
+import { call, put, select, takeEvery, takeLatest } from 'redux-saga/effects';
+import { ActionType } from 'typesafe-actions';
+import * as actions from '../actions';
+
+import * as actionTypes from '../actions/actionTypes';
+import { IState } from '../reducers/states';
+import { showSuccessMessage, showWarningMessage } from '../utils/notification';
+
+let initialized = false;
+
+const CLIENT_ID = '352013967309-qmhsu2ddofc79hpa0r3fe4q2djl16d7m.apps.googleusercontent.com';
```

Comment:
why declare the the client id again? Why not just take from .env? Also its super bad practice to hardcode credentials in source files!

Reply:
fixed